lib/src/create/presenter/widgets/bottom_bar.dart

```diff
@@ -75,7 +75,23 @@ class BottomBar extends StatelessWidget {
             ),
             IconButton(
               icon: const Icon(Icons.camera_alt_outlined),
-              onPressed: () => _takePhoto(context),
+              onPressed: () => _takePhoto(context),       
+            ),
+            IconButton(
+              icon: const Icon(Icons.gif_box_outlined), 
+              onPressed: () {
```

Comment:
Me parece que en este onPressed sería mejor crear un método _pickGifFromTenor o algo similar, así es más fácil de leer y mantener el código 

```suggestion
              onPressed: () => _pickGifFromTenor(context),
```